container/src/services/webcomponents.service.ts

```diff
@@ -332,6 +332,9 @@ export class WebComponentService {
       getUserSettings: (): Object => {
         return this.thisComponent.userSettings || {};
       },
+      getSkipInitCheck: (): boolean => {
+        return !!this.thisComponent.skipInitCheck;
+      },

```

Comment:
I am not really sure if we need to have any `getSkipInitCheck` function. We don't have it for Container as well as Core WC. 
The `skip-init-check` is not a value that is set on the Container and we can fetch it from the inside. Its more a boolean that will affect how the component starts - with or without waiting for Init Check.